tests/frontend/onnx/test_context.py

```diff
@@ -0,0 +1,191 @@
+import onnx
+import pytest
+
+try:
+    from onnx import (
+        TensorProto,
+        ValueInfoProto,
+        helper,
+    )
+
+    from xdsl.frontend.onnx.context import (
+        OnnxXdslMapping,
+        build_module,  # noqa: E402
+        visit_graph,  # noqa: E402
+        visit_node,  # noqa: E402
+        visit_value_info,  # noqa: E402
```

Comment:
can you please remove noqa from these lines?

Reply:
Done